src/Avalonia.Controls/Automation/Peers/AutomationPeer.cs

```diff
@@ -0,0 +1,259 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using Avalonia.Automation.Platform;
+
+#nullable enable
+
+namespace Avalonia.Automation.Peers
+{
+    public enum AutomationControlType
```

Comment:
Yeah this list comes from the WPF API. Should we just replace this list with values from the ARIA roles if that's the common subset? Should we keep the WPF/win32 naming or use the ARIA naming? Perhaps that latter would make more sense from an x-plat point of view.

Reply:
I think sticking with the WPF/Win23/UWP naming is easier since folks familiar with XAML are more familiar with those names. Developers coming from the Web will have to learn XAML first anyway, having some web naming in here might seem weird then.